gradle-hsql-plugin/gradle.properties

```diff
@@ -0,0 +1 @@
+serverbase = /home/stalele/gradleTomcat
```

Comment:
Should `gradle.properties` come missing and require the adopter to create it working from a `grade.properties.example`?